### PR TITLE
Fix #392: Remove V2 API support

### LINKED ISSUE #392: Remove V2 API support
In 1.5.x, we definitely removed the legacy V2 protocol support and only support versions 3.x and newer.

### PR DESCRIPTION
**Pull request by banterCZ**

### CODE REVIEW

File: powerauth-restful-model/src/main/java/io/getlime/security/powerauth/rest/api/model/request/ActivationLayer1Request.java [hunk 1]
```diff
@@ -17,7 +17,7 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-package io.getlime.security.powerauth.rest.api.model.request.v3;
+package io.getlime.security.powerauth.rest.api.model.request;
```
Comment (banterCZ) [inline powerauth-restful-model/src/main/java/io/getlime/security/powerauth/rest/api/model/request/ActivationLayer1Request.java:20]: 🤔 Do we want to keep the package for the version (for example when we introduce `v4`)?

Comment (hvge) [reply powerauth-restful-model/src/main/java/io/getlime/security/powerauth/rest/api/model/request/ActivationLayer1Request.java:20]: Yes, there are some minor differences between `v3` and future `v4` model. I would keep `v3` for now and we'll see whether it's efficient to cover v3 as a subset of v4 API or we'll do the full separation.

Comment (banterCZ) [reply powerauth-restful-model/src/main/java/io/getlime/security/powerauth/rest/api/model/request/ActivationLayer1Request.java]: Ok, I have reverted the commit with `v3` removal.

Comment (banterCZ) [reply powerauth-restful-model/src/main/java/io/getlime/security/powerauth/rest/api/model/request/ActivationLayer1Request.java]: Once again, let's drop the `v3` package.

**Review by romanstrobl — APPROVED** [review]
Looks OK.

**Pull request merged**